### PR TITLE
Add --no-round argument to use designspace roundGeometry=False

### PR DESCRIPTION
**Pull request by moyogo**
See https://github.com/googlei18n/glyphsLib/pull/220

### CODE REVIEW

File: Lib/fontmake/font_project.py [hunk 1]
```diff
@@ -413,12 +417,14 @@ def run_from_designspace(
 
         ufos = []
         if not interpolate or masters_as_instances:
-            reader = DesignSpaceDocumentReader(designspace_path, ufoVersion=3)
+            reader = DesignSpaceDocumentReader(designspace_path, ufoVersion=3,
+                                               round_geometry=round_geometry)
```
Comment (anthrotype) [inline Lib/fontmake/font_project.py]: I believe here passing `round_geometry` argument doesn't make any difference, as no interpolation is happening in this branch.

Comment (moyogo) [reply Lib/fontmake/font_project.py]: Good point.

**Comment by moyogo** [comment]
@anthrotype ping

File: Lib/fontmake/__main__.py [hunk 2]
```diff
@@ -87,6 +87,9 @@ def main(args=None):
     outputGroup.add_argument(
         '--family-name',
         help='Family name to use for masters, and to filter output instances')
+    outputGroup.add_argument(
+        '--no-round', dest='no_round', action='store_true',
+        help='Do not round to integers in ouput when interpolating.')
```
Comment (anthrotype) [inline Lib/fontmake/__main__.py]: `s/ouput/output/`

**Pull request merged**